### ISSUE TITLE
Det tar lang tid å få bilder fra asperøy.no

### ISSUE THREAD

**Issue by eibakke** [gudbrandtandberg/Asperoy#5]
Ser at det tar veldig lang tid å laste inn bilder fra serveren vår. Ifølge Chrome consollen tar det opp til 20 ganger så lang tid for noen bilder i forhold til resten av siden.

Er ikke helt 100% sikker, men tror nok det har med bildestørrelsen å gjøre.
[Her er bildet som tok lengst tid](http://xn--aspery-fya.no/resources/bilder/karusellbilder/IMG_1765.jpg)

Trenger dette bildet strengt tatt å være 2595x1936 pixler? Jeg foreslår at vi legger til:
1. Restriksjoner på bildestørrelse (et visst antall kBs)
2. Restriksjoner på dimensjoner (et visst antall pixler)

Vi kan lagre bildene som strenger eller filer, men hvis vi forminsker dem kan vi nok gjøre det hele mye raskere.

[Sjekk ut dette fungerende eksempelet på bildekomprimeringen vi kan bruke for nye opplastninger, og kanskje også de eksisterende bildene vi har](http://makeitsolutions.com/labs/jic/)

**Comment by gudbrandtandberg** [comment]
Ja!  Jeg er et skritt foran. Eller ved siden av hvertfall! sjekk ut master i Hildesiden. Der har jeg delvis implementert filhåndtering og opplastning (js funker men php ikke ennå). Kan nok bruke mye av koden igjen (eller bygge videre på). Er også et shell script vi kan bruke til å komprimere de lokale bildene vi har en gang for alle (jeg gjør det nå og pusher). Alle bildene som er der nå er jo bare test-bilder da.. 

Tenkte nettop i dusjen på hvordan vi skal komprimere. Input på være dimensjon og filstørrelse (eg 1000x1024px 4MB). Så bestemmer vi oss for en viss resulosjon vi ønsker (eg 90 dpi). En formel gir oss kompresjonsraten (eg 90%).  Passe skækk nå...

**Comment by gudbrandtandberg** [comment]
## Finnes det en måte å diskutere ting som ikke er problemer på GH?

Jeg bare lurer. Kan vi bruke planechat tror du? 

### Jeg liker Domeneshop.
- Fordi de er størst
- Fordi de holder til i Norge
- Fordi de tilbyr mange viktige tjenester (epost @asperøy.no, DNS-videresending: flere domener kan sendes til serveren (eivind.no, duffdev.io, ..)
- De er enkle å ha med å gjøre

Prøv selv, gå inn på http://www.domeneshop.no og logg inn med brukernavn asperøy. Du får passord i kryptert mail. 

#### Kanskje et godt sted å diskutere her

Det gjelder bare å kunne __itt_ markdown, så går det nok __veldig__ bra. 

´Peace Out´

**Comment by gudbrandtandberg** [comment]
Tips om ikke du allerede har:

.bashprofile <<
alias asp="cd ~/Der/Hvor/Asperøy/Er"
alias hld="cd ~/Der/Hilde/Er"

**Comment by gudbrandtandberg** [comment]
#### For å bruke compress.sh.

Advarsel: **den komprimerer alle jpger du har** med imagemagick sin convert funksjon _hvis du har den da!_.

> > sudo cp compress.sh /usr/bin
> > cd targetdir
> > sudo compress

**Comment by gudbrandtandberg** [comment]
##usexy

![screen shot 2015-03-03 at 22 40 52](https://cloud.githubusercontent.com/assets/6566727/6472976/656a60d6-c1f6-11e4-99c0-e3a783c2545a.png)
![screen shot 2015-03-03 at 22 40 31](https://cloud.githubusercontent.com/assets/6566727/6472977/656ac684-c1f6-11e4-92fb-ba12f204bac3.png)
![screen shot 2015-03-03 at 22 40 41](https://cloud.githubusercontent.com/assets/6566727/6472978/656f0e92-c1f6-11e4-90e1-c15962f030a8.png)

**Comment by gudbrandtandberg** [comment]
Det neste jeg skal gjøre når jeg begynner arbeidet på siden er å skrive om til å bruke bootstrap. Syns det var ganske luksus å slippe å bry seg om så mye pirk. Men vi custumizer det **to tha max** så blir det _smukt_.

**Comment by gudbrandtandberg** [comment]
# NATTA
